lition/README.md

```diff
@@ -0,0 +1,28 @@
+# Chainlink External Adapter for Lition
+
+## Price API
+
+### Endpoint
+
+https://staking.lition.io/api/v1/energy/source/1/date/2020-10-07/hour/15/
+
+### Input Params
+
+- `source`: The provider to retrieve price data from (required, e.g. "1", "2", or "3")
```

Comment:
Do you think it is better to leave the param as an integer or just have the param be a string of the source's name?

Reply:
It may be better to leave the numbers, as their API expects it as such, so if they add additional sources we do not need to update the adapter but just use the different source id.

Ideally, we would just link to their documentation where they list all the sources, if they have something like that available.

Reply:
I haven't been able to find any documentation about the sources, but added to the README for what I was able to uncover via the API.